src/ontology/cl-edit.owl

```diff
@@ -27978,11 +27981,12 @@ SubClassOf(obo:CL_4023040 obo:CL_4023008)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.343129") obo:IAO_0000115 obo:CL_4023041 "A glutamatergic neuron, with a soma found in the deeper portion of L5, that has long-range axonal projections including deep subcortical targets outside of the telencephalon and, in some cases, the spinal cord. While the L5 ET neuron projections are not limited to ET targets, they are clearly differentiated from the neuron subclasses whose projections are constrained to intratelencephalic (IT) targets.  L5 ET neurons are generally the largest excitatory cortical neurons, typically having a thick apical dendrite with a prominent dendritic tuft in layer 1 and displaying burst-firing physiological characteristics.")
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023041 <http://orcid.org/0000-0001-7258-9596>)
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023041 "L5b neuron")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023041 "burst-firing layer 5 neuron")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023041 "pyramidal tract (PT) neuron")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023041 "subcerebral projection (SCPN) neuron")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023041 "thick-tufted layer 5 pyramidal neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/j.neuron.2011.07.029") oboInOwl:hasExactSynonym obo:CL_4023041 "L5b neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/j.cell.2019.09.023") Annotation(rdfs:comment "pyramidal tract-like is used because, unlike pyramidal tract neurons in the motor cortex, neurons in the structures like the auditory cortex do not project to the spinal cord.") oboInOwl:hasExactSynonym obo:CL_4023041 "Pyramidal tract-like (PT-l)")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.3389/fncel.2011.00001") oboInOwl:hasExactSynonym obo:CL_4023041 "burst-firing layer 5 neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1038/nrn3469") oboInOwl:hasExactSynonym obo:CL_4023041 "pyramidal tract (PT) neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/j.neuron.2005.08.030") oboInOwl:hasExactSynonym obo:CL_4023041 "subcerebral projection (SCPN) neuron")
```

Comment:
I think this one should be a broad synonym:
![image](https://user-images.githubusercontent.com/112839/123770800-7ac64000-d8c2-11eb-974e-4606d235760c.png)